sql/scan.go

```diff
@@ -673,15 +674,22 @@ func (n *scanNode) getQVal(col ColumnDescriptor) parser.Expr {
 
 type qnameVisitor struct {
 	*scanNode
-	err error
+	stack []parser.Expr
```

Comment:
what's the relationship between the length of `stack` and the value of `pre` passed into `Visit`, if one exists